src/hand-tracking/src/main.cpp

```diff
@@ -145,9 +144,9 @@ int main(int argc, char *argv[])
     yInfo() << log_ID << " - q_x:"        << paramsd["q_x"];
     yInfo() << log_ID << " - q_y:"        << paramsd["q_y"];
     yInfo() << log_ID << " - q_z:"        << paramsd["q_z"];
-    yInfo() << log_ID << " - theta:"      << paramsd["theta"];
-    yInfo() << log_ID << " - cone_angle:" << paramsd["cone_angle"];
-    yInfo() << log_ID << " - seed:"       << paramsd["seed"];
+    yInfo() << log_ID << " - q_phi:"      << paramsd["q_phi"];
```

Comment:
I would prefer to have `roll`, `pitch` and `yaw` as I find them more clear. What do you think?

Reply:
I agree on it. 

I will change them according to:
- `q_phi` -> `yaw`
- `q_theta` -> `pitch`
- `q_psi` -> `roll`

as I am using a Euler ZYX parametrization.